posts/app.py

```diff
@@ -60,28 +61,52 @@ def upload_new_post():
 @app.route('/v1/', methods=['GET'])
 def get_all_posts():
     """Get all posts for the whole event."""
-    # TODO(mukobi) change from fake data to real query from db
-    return jsonify({
-        "post_id": "abc123",
-        "event_id": "all",
-        "author_id": "mukobi",
-        "created_at": "2017-10-06T00:00:00+00:00",
-        "text": "This is the description of a post. The post is tagged with "
-                "the event id of 'all' and should be viewable by default.",
-        "images": [
-            "https://upload.wikimedia.org/wikipedia/commons/6/66/An_up-close_picture_of_a_curious_male_domestic_shorthair_tabby_cat.jpg",
-            "http://4.bp.blogspot.com/-w8U75TCuhgU/Tzw8TmaclvI/AAAAAAAABJ0/6fMMcRLAceM/s1600/Rabbit3.jpg"
-        ]})
+    # serialize otherwise nonserializable ObjectIDs
+    return json_util.dumps(find_posts_in_db(POSTS_COLLECTION))
```

Comment:
Do you need to load this into json again? At least for the response type to be recognized as json, not sure if it's automatic.

Reply:
Thanks, done in https://github.com/knative-portability/large-events/pull/24/commits/4000202c4c42a327ab14761ea5c3ad6a97f79d96 https://github.com/knative-portability/large-events/pull/24/commits/c67368b4aec6bf6aa196edf45f3a1f21527b2972 where previously it was returning text. I also had to wrap the list result of the query into a dict for valid json, so now responses look like ```
{javascript
    "posts": {
        [
            post1...,
            post2...
        ]
    },
    "num_posts": 2
}
```
as per the feature doc.